docs/reference/components/sources/host_metrics.cue

```diff
@@ -44,6 +44,24 @@ components: sources: host_metrics: {
 		platform_name: null
 	}
 
+    env_vars: {
+		PROCFS_ROOT: {
+			description: "Sets an arbitrary path to the system's Procfs root. Can be used to expose host metrics from within a container."
+			type: string: {
+				default: null
```

Comment:
Should this default be `"/proc"` to describe the directory normally used?

Reply:
So this was a consideration I also made but I think the answer is technically no? The documentation is for a envvar that's not normally set on a system so null is the default value. Maybe I ought to clarify that in the description?

Reply:
The same could be said for any optional value, but I don't think that's particularly useful. More useful would be to document the underlying value and what it defaults to if the setting is not present.